README.md

```diff
@@ -20,7 +20,7 @@ This software is EXPERIMENTAL software. USE ON MAINNET AT YOUR OWN RISK.
 
 By default this software will use your existing Bitcoin wallet, including spending bitcoins contained therein (for example for transaction fees or trading).
 The protocol and transaction processing rules for the Omni Layer are still under active development and are subject to change in future.
-Omni Core should be considered an alpha-level product, and you use it at your own risk. Neither the Omni Foundation nor the Omni Core developers assumes any responsibility for funds misplaced, mishandled, lost, or misallocated.
+Omni Core should be considered an beta-level product, and you use it at your own risk. Neither the Omni Foundation nor the Omni Core developers assumes any responsibility for funds misplaced, mishandled, lost, or misallocated.
```

Comment:
In case of doubt, let's stick to the status quo. If it's not consistent, then we can fix it later. I'm going to change those `bitcoin` to `omnicore`, and switch back to `alpha`. My primary goal was to fix the bad link, and clean up the obvious parts.

But we may re-evalute at some point, how we paint Omni Core. It's a bit strange, if nearly half a million tUSD are out there, and we warn at the same time that the software has an experimental, alpha-level status, and it isn't intended to be used on mainnet. (just to clarify: I'm in favor of being as careful as possible here!)